docs/ExpectAPI.md

```diff
@@ -930,6 +937,11 @@ describe('the La Croix cans on my desk', () => {
 
 > Note: `.toEqual` won't perform a _deep equality_ check for two errors. Only the `message` property of an Error is considered for equality. It is recommended to use the `.toThrow` matcher for testing against errors.
 
+If differences between properties do not help you to understand why a test fails, especially if the report is large, then you might move the comparison into the `expect` function. For example, use `equals` method of `Buffer` class to assert whether or not buffers contain the same content:
+
+- rewrite `expect(received).toEqual(expected)` as `expect(received.equals(expected)).toBe(true)`
```

Comment:
we could potentially suggest custom matchers so they can show the diff they want?